common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/activedirectoryfederationservices/ActiveDirectoryFederationServices2012R2OAuth2Strategy.java

```diff
@@ -106,7 +105,7 @@ public TokenRequest createTokenRequest(AuthorizationRequest request, Authorizati
     }
 
     @Override
-    public TokenRequest createRefreshTokenRequest(RefreshTokenRecord refreshToken, List scopes) {
+    public TokenRequest createRefreshTokenRequest(RefreshTokenRequestParameters parameters) {
```

Comment:
Is it necessary for the strategy to create this TokenRequest?  I'm wondering why the refresh token request is different than just a TokenRequest.